client/app/bundles/course/assessment/pages/AssessmentIndex/AssessmentIndexPage.tsx

```diff
@@ -0,0 +1,147 @@
+import { useMemo } from 'react';
+import {
+  AssessmentListData,
+  AssessmentsListData,
+} from 'types/course/assessment/assessments';
+
+import Link from 'lib/components/core/Link';
+import Table, { ColumnTemplate } from 'lib/components/core/table';
+import PersonalStartEndTime from 'lib/components/extensions/PersonalStartEndTime';
+import useTranslation from 'lib/hooks/useTranslation';
+
+import Actions from './Actions';
+import StackedBadges from './StackedBadges';
+import StatusBadges from './StatusBadges';
+import translations from './translations';
+
+interface AssessmentIndexProps {
+  assessments: AssessmentsListData;
+  top: string;
+}
+
+const AssessmentIndexPage = (props: AssessmentIndexProps): JSX.Element => {
+  const { display, assessments } = props.assessments;
+  const { t } = useTranslation();
+
+  const template: ColumnTemplate<AssessmentListData>[] = useMemo(
+    () => [
+      {
+        header: t(translations.title),
+        content: (assessment) => (
+          <div className="flex flex-col items-start justify-between xl:flex-row xl:items-center">
+            <label className="m-0 font-normal" title={assessment.title}>
+              <Link
+                className="line-clamp-1"
```

Comment:
What's your thought on displaying multiline title when the screen size is small or hiding `ends at`?
If I am the user, it feels troublesome if i need to rotate my phone..
![image](https://user-images.githubusercontent.com/42570513/202613760-ebba72bf-d520-4f58-966f-33e9545be36a.png)

Reply:
I also think it is a good idea. I have increased the number of lines to two for smaller screens.

Actually, _Ends at_ is hidden only on _mobile_ screens ≤ `sm`.